CMakeLists.txt

```diff
@@ -31,12 +31,19 @@ if(NOT ${CMAKE_SOURCE_DIR} STREQUAL ${PROJECT_SOURCE_DIR} OR
 endif()
 
 common_find_package(Boost REQUIRED COMPONENTS program_options unit_test_framework)
-common_find_package(OSPRay REQUIRED SYSTEM)
-common_find_package(GLUT REQUIRED SYSTEM)
-common_find_package(GLEW REQUIRED SYSTEM)
-common_find_package(OpenGL REQUIRED)
 common_find_package(vmmlib REQUIRED)
 
+option(BRAYNS_OSPRAY_ENABLED "Activate OSPRay engine" ON)
+if(BRAYNS_OSPRAY_ENABLED)
+  common_find_package(OSPRay REQUIRED SYSTEM)
+endif()
+
+option(BRAYNS_OPTIX_ENABLED "Activate OptiX engine" OFF)
+if(BRAYNS_OPTIX_ENABLED)
+  common_find_package(OptiX REQUIRED SYSTEM)
+  common_find_package(CUDA REQUIRED SYSTEM)
+endif()
```

Comment:
Hum... ok, changed because I understand this is what we do everywhere else. But to me, it's not because the dependencies are present that you necessarily want to build the module.